include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -175,7 +207,7 @@ class BCX_API qrcode
         )
         (
             "filename",
```

Comment:
For consistency with other commands (and the avoidance of the compound name) this should be `IMAGE_FILE`. Note that **arguments** are upper case by bx convention (and **options** are lower case).